components/pedigree/resources/src/main/resources/pedigree/model/dynamicGraph.js

```diff
@@ -1853,13 +1854,27 @@ define([
                 importData.unlinkedMembers = {};
             }
 
-            // FIXME: for now, we do not allow linking patients via import dialogue, so remove all links to all patients
+            // TODO: for now, we do not allow linking patients via import dialogue, so remove
+            //       all links to all patients records that are not laready in the family
             if (importOptions && importOptions.doNotLinkPatients) {
-                unlinkedMembers = [];
+                // remove all unlinked patients loaded form import data that are not already in family:
+                for (var patientRecordId in importData.unlinkedMembers) {
+                    if (importData.unlinkedMembers.hasOwnProperty(patientRecordId)) {
+                        if (!editor.isFamilyMember(patientRecordId)) {
```

Comment:
why not use && to combine two `if`s?

Reply:
to distinguish between technical `.hasOwnProperty()` check for object properties that has to be done in javascript, and actual application logic. If `hasOwnProperty` were applied because of application logic I would combine the two